### DISCUSSION TITLE
[Announcement] holidays & plans

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs524-program-analysis#394] [Announcements]
Hi all,

This semester especially has many holidays on Mon and Wed, so that we are in a little bit of trouble.
So I want to propose to have all the remaining lectures for 1.5hrs (until 10:30 am). I think it is better than having additional lectures separately which might open another complexity. 

Let me know if you have any problem.

**Comment by ShinWonho** [comment]
안녕하세요, CS592 수업 수강 중인 신원호 입니다.

제가 10월 6일 10시 30분에 CS376 시험이 있습니다.

그래서 만약 10시 30분까지 수업을 하시게 된다면, 이 날은 조금 일찍 나가봐야 할 것 같습니다.

그 외에 날에는 10시 30분까지 수업을 하시는데 문제가 없습니다.

감사합니다.

신원호 올림.



-----Original Message-----
From: "Kihong Heo" ***@***.***&gt;
To: "prosyslab-classroom/cs592-2021-fall" ***@***.***&gt;;
Cc: "Subscribed" ***@***.***&gt;;
Sent: 2021-09-30 (목) 15:45:18 (UTC+09:00)
Subject: [prosyslab-classroom/cs592-2021-fall] [Announcement] holidays &amp; plans (#28)


Hi all,
This semester especially has many holidays on Mon and Wed, so that we are a little bit of trouble.So I want to propose to have all the remaining lectures for 1.5hrs (until 10:30 am). I think it is better than having additional lectures separately which might open another complexity.
Let me know if you have any problem.

**Comment by KihongHeo** [comment]
원호씨,

네 알겠습니다. 편할때 일찍 나가시고, 뒷부분은 동영상을 올릴테니 보시면 되겠습니다.

고맙습니다.
- 허기홍 드림

**Comment by ShinWonho** [comment]
메일 확인했습니다.

감사합니다!!
신원호 올림.-----Original Message-----From: Kihong Heo ***@***.***&gt;To: prosyslab-classroom/cs592-2021-fall ***@***.***&gt;Cc: 신원호 ***@***.***&gt;, Comment ***@***.***&gt;Sent: 2021-09-30T16:19:54+09:00Subject: Re: [prosyslab-classroom/cs592-2021-fall] [Announcement] holidays &amp; plans (#28)


원호씨,



네 알겠습니다. 편할때 일찍 나가시고, 뒷부분은 동영상을 올릴테니 보시면 되겠습니다.



고맙습니다.

- 허기홍 드림



&gt; 9/30/21 오후 4:18, ShinWonho ***@***.***&gt; 작성:

&gt; 

&gt; 

&gt; 안녕하세요, CS592 수업 수강 중인 신원호 입니다.

&gt; 

&gt; 제가 10월 6일 10시 30분에 CS376 시험이 있습니다.

&gt; 

&gt; 그래서 만약 10시 30분까지 수업을 하시게 된다면, 이 날은 조금 일찍 나가봐야 할 것 같습니다.

&gt; 

&gt; 그 외에 날에는 10시 30분까지 수업을 하시는데 문제가 없습니다.

&gt; 

&gt; 감사합니다.

&gt; 

&gt; 신원호 올림.

&gt; 

&gt; 

&gt; 

&gt; -----Original Message-----

&gt; From: "Kihong Heo" ***@***.***&amp;gt;

&gt; To: "prosyslab-classroom/cs592-2021-fall" ***@***.***&amp;gt;;

&gt; Cc: "Subscribed" ***@***.***&amp;gt;;

&gt; Sent: 2021-09-30 (목) 15:45:18 (UTC+09:00)

&gt; Subject: [prosyslab-classroom/cs592-2021-fall] [Announcement] holidays &amp;amp; plans (#28)

&gt; 

&gt; 

&gt; Hi all,

&gt; This semester especially has many holidays on Mon and Wed, so that we are a little bit of trouble.So I want to propose to have all the remaining lectures for 1.5hrs (until 10:30 am). I think it is better than having additional lectures separately which might open another complexity.

&gt; Let me know if you have any problem.

&gt; —You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub https://github.com/prosyslab-classroom/cs592-2021-fall/issues/28, or unsubscribe https://github.com/notifications/unsubscribe-auth/AL5TQR2TK6L7NNFJKC3U3NTUEQBPZANCNFSM5FBRXEQA.Triage notifications on the go with GitHub Mobile for iOS https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;amp;mt=8&amp;amp;pt=524675 or Android https://play.google.com/store/apps/details?id=com.github.android&amp;amp;referrer=utm_campaign=notification-email&amp;amp;utm_medium=email&amp;amp;utm_source=github. 

&gt; —

&gt; You are receiving this because you authored the thread.

&gt; Reply to this email directly, view it on GitHub &lt;https://github.com/prosyslab-classroom/cs592-2021-fall/issues/28#issuecomment-930908908&gt;, or unsubscribe &lt;https://github.com/notifications/unsubscribe-auth/AAJY4YOFJZWEEOSUBS4YV3DUEQFLRANCNFSM5FBRXEQA&gt;.

&gt;

**Comment by AnHaechan** [comment]
안녕하세요, 20180368 안해찬입니다.
저도 신원호님과 마찬가지로 10월 6일 10시 30분에 다른 수업 시험이 있어서, 일찍 수업을 나가야 할 거 같습니다.
감사합니다.

안해찬 드림
